routes/users.js

```diff
@@ -26,4 +25,54 @@ module.exports = function (app, middelwares) {
       return next(err);
     }
   }
-};
\ No newline at end of file
+
+  //Add new user
+  routerUsers.post("/", createUser);
+  async function createUser(req, res, next) {
+    try {
+      const body = req.body;
+      const results = await app.bll.users.createUser(body);
+
+      return res.status(200).send(results);
+    } catch (err) {
+      return next(err);
+    }
+  }
+
+  //Get user
+  routerUsers.get("/:id", getUserById);
+  async function getUserById(req, res, next) {
+    try {
+      const id = req.params.id;
+      const results = await app.bll.users.getUserById(id);
+      return res.status(200).send(results);
+    } catch (err) {
+      return next(err);
+    }
+  }
+
+  //Update user
+  routerUsers.patch("/:id", updateUser);
```

Comment:
Nên dùng PUT, nếu dùng PATCH thì trong route cần lấy user cũ ra để merge lại với body.
```suggestion
  routerUsers.put("/:id", updateUser);
```